### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by hansl**
This is still early, but it'd be nice to get this in while I'm moving to writing tutorial on Schematics.

### CODE REVIEW

File: .monorepo.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        "url": "[david-badge-url]"
+      },
+      {
+        "title": "![devDependency Status][david-dev-badge]",
+        "url" : "[david-dev-badge-url]"
+      }
+    ], [
+      {
+        "label": "npm",
+        "image": "https://img.shields.io/npm/v/%40angular/cli.svg",
```
Comment (filipesilva) [inline .monorepo.json]: A lot of these are still for the CLI.

File: lib/packages.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import * as ts from 'typescript';
 
 const glob = require('glob');
 const distRoot = path.join(__dirname, '../dist');
+const { packages: monorepoPackages } = require('../.monorepo.json');
```
Comment (filipesilva) [inline lib/packages.ts]: 👍

File: packages/angular_devkit/core/src/utils/template.ts [hunk 3]
```diff
@@ -143,6 +155,12 @@ export function templateParser(sourceText: string, fileName: string): TemplateAs
       children.push({ kind: 'escape', expression: escape, start, end } as TemplateAstEscape);
       start = end;
     }
+    if (comment) {
+      end = _positionFor(sourceText, offset + comment.length + 5);
```
Comment (filipesilva) [inline packages/angular_devkit/core/src/utils/template.ts]: What's the meaning of `5` here? Just curious, I see other cases around this one also use it.

File: scripts/release.ts [hunk 4]
```diff
@@ -123,8 +119,8 @@ export default function(args: { _: string[], 'dry-run'?: boolean }, logger: Logg
     case 'patch':
       _upgrade(maybeRelease, logger);
       if (!dryRun) {
-        fs.writeFileSync(path.join(__dirname, '../versions.json'),
-                         JSON.stringify({ versions, hashes }, null, 2) + '\n');
+        fs.writeFileSync(path.join(__dirname, '../.monorepo.json'),
```
Comment (filipesilva) [inline scripts/release.ts]: Won't this delete all other fields in `monorepo.json` except for `packages`?

File: scripts/templates/readme.ejs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+  Any changes to README.md directly will result in a failure on CI.
+-->
+
+# Angular DevKit
+### Development tools and libraries specialized for Angular
+
+<%# Badges are created here. %>
+<% for (const section of monorepo.badges) {
+  for (const { githubBadge, label, image, title, url } of section) {
```
Comment (filipesilva) [inline scripts/templates/readme.ejs]: `githubBadge` is unused.

File: packages/angular_devkit/core/src/utils/template.ts (hunk 3 above)
Comment (hansl) [reply packages/angular_devkit/core/src/utils/template.ts]: Number of characters taken from the template itself (the `<%?` and `%>` parts).

File: .monorepo.json (hunk 1 above)
Comment (hansl) [reply .monorepo.json]: done.

File: scripts/release.ts (hunk 4 above)
Comment (hansl) [reply scripts/release.ts]: Fixed, you were right. Thanks for catching that!

File: scripts/templates/readme.ejs (hunk 5 above)
Comment (hansl) [reply scripts/templates/readme.ejs]: Removed.

**Pull request merged**